sdk/keyvault/keyvault-certificates/src/core/keyVaultClient.ts

```diff
@@ -37,43 +32,26 @@ class KeyVaultClient extends KeyVaultClientContext {
    * @param [options] The optional parameters
    * @returns Promise<Models.GetCertificatesResponse>
    */
-  getCertificates(
-    vaultBaseUrl: string,
-    options?: Models.KeyVaultClientGetCertificatesOptionalParams
-  ): Promise<Models.GetCertificatesResponse>;
+  getCertificates(vaultBaseUrl: string, options?: Models.KeyVaultClientGetCertificatesOptionalParams): Promise<Models.GetCertificatesResponse>;
```

Comment:
You must be having a different formatter running.. 
Can you look into what made these changes? Otherwise we will have multiple PRs that keep making these changes back and forth based on who is sending the PR

Reply:
No I did not format the code. They were straight from codegen. I don't think codegen respects any formatter configs.

It looks like the generated files were touched manually in https://github.com/Azure/azure-sdk-for-js/pull/5494/commits/f36e3fdef4bc72ace54879659793fa2bc7066bc2 and probably got formatted.

Reply:
Ah! In Keyvault the generated code is under "core". I expected to see them under "generated" and so thought this was a hand written file

Reply:
I think we have a tracking issue to rename `core` to `generated`. Also yeah we should exclude generated code from formatting.